TYPO3.Flow/Classes/TYPO3/Flow/Error/Debugger.php

```diff
@@ -175,14 +180,14 @@ protected static function renderObjectDump($object, $level, $renderProperties =
         $scope = '';
         $additionalAttributes = '';
 
-        if ($object instanceof \Doctrine\Common\Collections\Collection) {
-            return self::renderArrayDump(\Doctrine\Common\Util\Debug::export($object, 3), $level, $plaintext, $ansiColors);
+        if ($object instanceof Collection) {
+            return self::renderArrayDump(Debug::export($object, 3), $level, $plaintext, $ansiColors);
```

Comment:
I think it's fine, it's a different class name and the general naming makes a lot of sense here. If yuo want to check which specific class it is you can jump to it anyway.